ZelBack/src/services/appsService.js

```diff
@@ -1062,6 +1062,37 @@ async function getAppFolderSize(appName) {
   }
 }
 
+/**
+ * Returns total app container storage in bytes
+ * @param {string} appName - name or id of the container
+ * @returns {Promise<number>}
+ */
+async function getContainerStorage(appName) {
+  try {
+    const containerInfo = await dockerService.dockerContainerInspect(appName, { size: true });
+    let containerTotalSize = +containerInfo.SizeRootFs ?? 0;
+    // traverse mounts/volumes to find size used on host
+    if (containerInfo?.Mounts?.length) {
+      containerInfo.Mounts.forEach((mount) => {
+        let source = mount?.Source;
+        if (source) {
+          // remove /appdata to get true size of the app folder
+          source = source.replace('/appdata', '');
+          const mountInfo = nodecmd.runSync(`sudo du -sb ${source}`);
```

Comment:
u should use async function for send shell cmd and other type of loop forEach not support async

Reply:
if u want get only mount volume size better use df ( it is already used df js lib in fluxos ), sudo du take long time and resources

Reply:
I can convert it to use cmdAsync and use a different iterator with async support.

I think du is the more accurate way to get the size of the directory, which is what is currently being used in the app monitoring solution but we can certainly look into doing this another way.

Reply:
but u dont need check any folder size as bind mount is volume u need check only volume size

Reply:
there is already size, used, and available info from df